active/ThemingCornerRadii/CornerRadiusAPI.md

```diff
@@ -0,0 +1,175 @@
+<!-- The purpose of this spec is to describe a new feature and
+its APIs that make up a new feature in WinUI. -->
+
+<!-- There are two audiences for the spec. The first are people
+that want to evaluate and give feedback on the API, as part of
+the submission process.  When it's complete
+it will be incorporated into the public documentation at
+docs.microsoft.com (http://docs.microsoft.com/uwp/toolkits/winui/).
+Hopefully we'll be able to copy it mostly verbatim.
+So the second audience is everyone that reads there to learn how
+and why to use this API. -->
+
+# Background
+<!-- Use this section to provide background context for the new API(s)
+in this spec. -->
+
+<!-- This section and the appendix are the only sections that likely
+do not get copied to docs.microsoft.com; they're just an aid to reading this spec. -->
+
+<!-- If you're modifying an existing API, included a link here to the
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+
+> See [proposal document](https://github.com/Microsoft/microsoft-ui-xaml/issues/684) for information on this feature's scope, and see the [parent proposal document](https://github.com/microsoft/microsoft-ui-xaml/issues/524) for information about design application of this feature.
+
+# Description
+<!-- Use this section to provide a brief description of the feature.-->
+The ``CornerRadius`` property for controls and their templates can be easily customized/themed through the introduction of new [Lightweight styling](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/xaml-styles#lightweight-styling) resources.
+
+# Examples
+The following example cover common use-case scenarios where an app-author would like to customize the corner radii of one or more controls.
+
+## Setting the CornerRadii of controls
+To make large changes globally and consistently to all controls within a scope, defining/overriding the ``ControlThemeCornerRadius`` and ``OverlayThemeCornerRadius`` resources from generic.xaml will affect all controls that benefit from rounded edges.
+
+``` xml
+<Application.Resources>
+  <Thickness x:Key="ControlThemeCornerRadius">10</Thickness>

```

Comment:
Why do we need the word "Theme" in these resource names?

Reply:
I was following a trend I noticed with thickness values in generic.xaml.

Reply:
Thicknesses tend to vary, usually between HC and light/dark themes. I'm assuming the corner radii won't change with theme, will they?

Reply:
I don't think so. In that case I can remove the ``theme`` part.